### ISSUE TITLE
PlasmaBlob class

### ISSUE THREAD

**Issue by StanczakDominik** [PlasmaPy/PlasmaPy#156]
This was mentioned by @lemmatum in #86, or at least that reminded me of the idea.

The current Plasma class is inherently a spatial object. That might be too much information for some analyses.

How about a single Plasma object that takes a bunch of parameters (this would require specifying a sufficiently minimal set of parameters, like temperature, density, maybe having multiple ways to specify that via optional keyword arguments etc) and is able to call methods from `physics.parameters` to quickly spit out other parameters you can calculate from that?

@lemmatum also mentions being able to figure out the regime a plasma is in from that data. I'm not 100% sure how that'd work yet, but it is a neat idea.

**Comment by lemmatum** [comment]
I think it should relatively easy to create the class and then add methods to the class for each important dimensionless parameter which tests the plasma regime. Then there could be one method which just calls all these other methods and prints a report. There would essentially be a print statement for each parameter that returns the value obtained and what this means for the regime of the plasma.

This is an example of the method I am thinking of:
```
def regime(self):
    degen_theta = self.degeneracy_parameter()
    if degen_theta >= 1:
        print(f"Theta = {degen_theta}. Plasma is non-degenerate.")
    elif degen_theta < 1:
        print(f"Theta = {degen_theta}. Plasma is degenerate.")
        
```

**Comment by samurai688** [comment]
I think something like this could be quite nice. It would provide kind of an object-oriented alternative to the individual plasma parameters functions.

I've always thought it would be cool to have a "tell me everything you can" function, so if you say, I make a plasma object and I only know the temperature and the species, let me know all the stuff you can calculate just based on that.

Could you combine it with the existing plasma object by making the spatial arguments optional? Might not be good if the current plasma object is envisioned more as part of a simulation.

**Comment by lemmatum** [comment]
@samurai688 The "tell me everything you can" function is exactly what I've had in mind! Especially as I've had the experience of ignoring a certain plasma parameter, thinking it'd be unimportant, and having it come back to bite me 😬 . Such a function would help avoid that scenario!

**Comment by StanczakDominik** [comment]
Closed by #193 with possible extension of more parameters to be added
